float32 torque1
float32 torque2
int32 port_num
